frontend-react/src/styles/_uswds_overrides.scss

```diff
@@ -465,3 +465,8 @@ figure {
         @include u-margin-top(2);
     }
 }
+
```

Comment:
We should remove this as it's a pretty big global change, should be resolved with `position="left"` above.